qiita_db/search.py

```diff
@@ -167,8 +167,8 @@ def __repr__(self):
 class QiitaStudySearch(object):
     """QiitaStudySearch object to parse and run searches on studies."""
 
-    # column names from required_sample_info table
-    required_cols = set(get_table_cols("required_sample_info"))
+    # column names from study_sample table
+    required_cols = set(get_table_cols("study_sample"))
```

Comment:
This check isn't required any more, since all we have in the table now is study id and sample id, correct?